astroquery/linelists/cdms/core.py

```diff
@@ -34,6 +33,74 @@ class CDMSClass(BaseQuery):
     TIMEOUT = conf.timeout
     MALFORMATTED_MOLECULE_LIST = ['017506 NH3-wHFS', '028528 H2NC', '058501 H2C2S', '064527 HC3HCN']
 
+    def __init__(self, fallback_to_getmolecule=False):
```

Comment:
Please add some basic docs for this

```suggestion
    def __init__(self, *, fallback_to_getmolecule=False):
```